msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -1898,16 +1898,20 @@ public String processPvtMsgSaveDraft() {
 
     PrivateMessage dMsg = null;
     if(getDetailMsg() != null && getDetailMsg().getMsg() != null && getDetailMsg().getMsg().getDraft()){
-    	dMsg =constructMessage(true, getDetailMsg().getMsg()) ;
+	    dMsg =constructMessage(true, getDetailMsg().getMsg()) ;
```

Comment:
space before the semicolon and after the equal

Reply:
Done!